templates/solutions/ubuntu-os/index.html

```diff
@@ -0,0 +1,653 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1X3osSqcm3rWLqYUJr5G4fXEEHCby-j5FEJwClLNx5S4/edit
+{% endblock meta_copydoc %}
+
+{% block title %}The world's favourite enterprise Linux platform{% endblock %}
+
+{% block meta_description %}
+  Ubuntu is the world's most popular enterprise Linux distribution. Unify your tech stack with the enterprise OS that is favoured for servers, IoT and applications.
+{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Innovate with confidence.</h1>
+        <h2>Choose the world's favourite enterprise Linux.</h2>
+        <p class="p-section--shallow">
+          Bring harmony to your stack from the desktop to the edge with Ubuntu, the OS that successful enterprises have in common. Ubuntu brings standardisation and stability  together, offering you a platform for innovation, combined with the freedom that transparent, open source code offers.
+        </p>
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="ubuntu-os-modal">Contact us <span class="u-off-screen">about Ubuntu OS solutions</span></a>
+      </div>
+      <div class="col">
+        <div class="p-image-container--16-9 is-highlighted">
+          <img class="p-image-container__image"
+               src="https://assets.ubuntu.com/v1/ec73e570-hero-img.png"
+               alt="" />
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>
+          Name the industry.
+          <br />
+          You'll find Ubuntu there.
+        </h2>
+      </div>
+      <div class="col">
+        <p>Enterprises that want to scale up and break new ground trust Ubuntu as the reliable, secure platform to do it.</p>
+        <p>
+          A 20-year track record of reliability and innovation means Ubuntu has left its footprints across industries, from finance to automotive to the edge of space.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-image-container--cinematic is-cover">
```

Comment:
```suggestion
      <div class="p-image-container--cinematic is-highlighted">
```
I think this should be `is-highlighted`, not `is-cover` per [this comment](https://github.com/canonical/canonical.com/pull/1314#issuecomment-2252956657)
![image](https://github.com/user-attachments/assets/dc246817-d943-4a60-8902-34d76f41b09c)